external-builds/pytorch/README.md

```diff
@@ -28,13 +28,32 @@ This incorporates advice from:
 
 ### Project and feature support status
 
-| Project / feature              | Linux support                                                                     | Windows support  |
-| ------------------------------ | --------------------------------------------------------------------------------- | ---------------- |
-| torch                          | ✅ Supported                                                                      | ✅ Supported     |
-| torchaudio                     | ✅ Supported                                                                      | ✅ Supported     |
-| torchvision                    | ✅ Supported                                                                      | ✅ Supported     |
-| Flash attention via [ao]triton | ✅ Supported                                                                      | ✅ Supported     |
-| FBGEMM GenAI                   | ❌ Not supported (see Issue [#2056](https://github.com/ROCm/TheRock/issues/2056)) | ❌ Not supported |
+> [!NOTE]
+> Project support status is defined by the following classifications
+>
+> 1. Supported: Official packages are distributed by the ROCm team
+> 1. Nightly: Included in TheRock-based packages, built nightly
+> 1. Community Supported: Enabled via community contribution
+> 1. Not Supported: Does not build, requires engineering to support
+
+> [!NOTE]
+>
+> | Package Index Domain            | Description                                                                                        |
+> | ------------------------------- | -------------------------------------------------------------------------------------------------- |
+> | https://download.pytorch.org/   | Official upstream releases with instructions at https://pytorch.org/get-started/locally/           |
+> | https://repo.amd.com/rocm/      | Stable releases distributed by AMD and built by TheRock (Built in https://github.com/ROCm/rockrel) |
+> | https://rocm.nightlies.amd.com/ | Nightly releases distributed by AMD and built by TheRock                                           |
+
+| Project / feature              | Linux support                                                                     | Windows support  | Package Index Domain            | Compatibility with TheRock |
+| ------------------------------ | --------------------------------------------------------------------------------- | ---------------- | ------------------------------- | -------------------------- |
+| torch                          | ✅ Supported                                                                      | ✅ Supported     | https://download.pytorch.org/   | ✅                         |
```

Comment:
i am not sure how true this is with the package index domain.

the torch we distribute via TheRock definitely integrates ROCm packages. So it is "rocm build with torch"
I know torch also provides "torch build with rocm" but this does (as far as I know) currently only include mainline (rocm 6/7.1?) and not rocm build by theRock.

considering that we are in theRock repo we might want to point rather to our build packages.

Also for pytorch releases we use rocm/pytorch which contains amd-specific changes compared to upstream release. only for nightly we use pytorch upstream.